src/components/shared/PayWarningModal.tsx

```diff
@@ -0,0 +1,38 @@
+import { Modal } from 'antd'
+import { t, Trans } from '@lingui/macro'
+import ExternalLink from 'components/shared/ExternalLink'
+
+export default function PayWarningModal({
+  visible,
+  onOk,
+  onCancel,
+}: {
+  visible: boolean
+  onOk: VoidFunction
+  onCancel: VoidFunction
+}) {
+  return (
+    <Modal
+      visible={visible}
+      onOk={onOk}
+      onCancel={onCancel}
+      okText={t`I understand`}
+      cancelButtonProps={{ hidden: true }}
+      width={400}
+    >
+      <h2>
+        <Trans>Heads up</Trans>
+      </h2>
+      <p style={{ fontWeight: 500 }}>
+        <ExternalLink href="https://github.com/jbx-protocol/juice-contracts">
+          <Trans>Juicebox contracts</Trans>
+        </ExternalLink>{' '}
+        <Trans>
```

Comment:
**suggestion**: let's wrap the whole text (including link) in a `Trans`, and remove the `Trans` on line 28. This way we're translating the whole sentence.

Reply:
100%